hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtil.java

```diff
@@ -802,6 +807,24 @@ public SingleProcessHBaseCluster startMiniCluster(StartTestingClusterOption opti
    */
   public SingleProcessHBaseCluster startMiniHBaseCluster(StartTestingClusterOption option)
     throws IOException, InterruptedException {
+
+    if (option.isExternalDFS()) {
+      assertNotNull("fs.defaultFS can not be null, if use external DFS", conf.get("fs.defaultFS"));
```

Comment:
In general I prefer we do not depend on junit methods in HBTU, as it will force our downstream users also use the same version of junit... But anyway, not your fault, I think we have already used it in this class...